src/main/kotlin/com/manimdsl/shapes/PythonStyle.kt

```diff
@@ -0,0 +1,46 @@
+package com.manimdsl.shapes
+
+class PythonStyle {
+    private val pythonStyleAttributes: MutableSet<PythonStyleAttribute> = mutableSetOf()
+
+    fun addStyleAttribute(pythonStyleAttribute: PythonStyleAttribute) {
+        pythonStyleAttributes.add(pythonStyleAttribute)
+    }
+
+    override fun toString(): String {
+        return if (pythonStyleAttributes.isNotEmpty()) {
+            ", ${pythonStyleAttributes.joinToString(", ")}"
+        } else ""
+    }
+}
+
+sealed class PythonStyleAttribute {
+    abstract val name: String
+    abstract val value: String
+
+    override fun toString(): String = "$name=$value"
+}
+
+sealed class ColorAttributePython : PythonStyleAttribute() {
+    private fun handleColourValue(): String {
+        return if (value.matches(Regex("#[a-fA-F0-9]{6}"))) {
+            // hex value
+            "\"${value}\""
+        } else {
+            // predefined constant
+            value.toUpperCase()
```

Comment:
I saw in the test.json that the colours were in upper case. This is actually handled here so the stylesheet can have lower case colours. This is just to mention in the documentation.